### PR TITLE
chore(xtask): Dont update run submodule update

### PR DESCRIPTION
**Pull request by The-personified-devil**
+ Misc small xtask improvements because people keep breaking the build and not looking where in the code the unwrap is

### CODE REVIEW

**Comment by The-personified-devil** [comment]
Oh I forgot this requires the CI fix...

**Review by zmerp — APPROVED** [review]
LGTM

**Comment by The-personified-devil** [comment]
I'll add `--depth=1` to the submodule clone since it's already huge and will only get larger if it does a full clone, so I'll rebase this and add it when the ci pr is merged.

**Comment by zmerp** [comment]
--depth=1 should not be needed because we define the submodule as shallow

**Comment by The-personified-devil** [comment]
fair enough, I'll still rebase tho

File: alvr/session/build.rs [hunk 1]
```diff
@@ -4,7 +4,7 @@ use std::{env, fmt::Write, fs, path::PathBuf};
 fn main() {
     let openvr_driver_header_string =
         fs::read_to_string(alvr_filesystem::workspace_dir().join("openvr/headers/openvr_driver.h"))
-            .unwrap();
+            .expect("Missing openvr header files, did you clone the submodule?\n");
```
Comment (zmerp) [inline alvr/session/build.rs:7]: Very minor, but I don't think you have to put a \n in the expect message

**Review by zmerp — COMMENTED** [review]
Overall LGTM

File: alvr/session/build.rs (hunk 1 above)
Comment (The-personified-devil) [reply alvr/session/build.rs:7]: Intentional, the line gets very long if the error is appended to the expect in the same line

Comment (zmerp) [reply alvr/session/build.rs:7]: Alright

**Review by zmerp — APPROVED** [review]

**Pull request merged**